app/celery/reporting_tasks.py

```diff
@@ -19,23 +19,27 @@
 
 @notify_celery.task(name="create-nightly-billing")
 @cronitor("create-nightly-billing")
-def create_nightly_billing(day_start=None):
+def create_nightly_billing(
+    day_start=None,
+    n_days=10,
```

Comment:
what does `n_days` mean?

Reply:
Number of days.

Reply:
If we had mathematical notation available to us this would be $N_{days}$ but we don't.

Reply:
:-)
probably not for this pr but I was confused with `n_days = 10 (number of days to process)` vs the comment `up to 4 days of data counting back from day_start is consolidated`

Reply:
I'll update it.